Packs/CommonPlaybooks/ReleaseNotes/2_3_25.md

```diff
@@ -0,0 +1,20 @@
+
+#### Playbooks
+##### New: User Investigation - SIEM search for failed logon - Generic
+- This playbook searches for failed logon on a specific user by querying logs from different sources.
```

Comment:
Let's change the playbook name to SIEM - Search for Failed logins

Reply:
Done.